models/cam-fv/work/input.nml

```diff
@@ -31,7 +41,7 @@
    init_time_days               = -1
    init_time_seconds            = -1
 
-   stages_to_write              = 'preassim','output'
+   stages_to_write              = 'forecast','output'
```

Comment:
one question on this change -- writing forecast will have un-inflated values, i believe, and typically cam uses prior inflation.  are you sure you don't want to leave this as preassim?   for the diagnostics we've always used preassim.

Reply:
You're right that in the past we used preassim and those have the inflated values.
But only the mean is used in the diagnostics (obs_diag), which is the same in preassim and forecast,
so it didn't matter in that context which was used.
There are times when a user wants to see the assimilation increments in the members instead of the mean.
Then they need to compare the posterior with an uninflated prior, which is forecast.
Otherwise, the increments will have the inflation in them, which is often much larger than the
assimilation increments and not physically meaningful.
We changed the stages in the midst of the Reanalysis to use forecast.
Based on all that I consider forecast to the best default.

Reply:
ah, ok - i didn't realize you'd changed to forecast.  so this comment is resolved and thanks for the explanation!